client/setec/store.go

```diff
@@ -241,6 +241,26 @@ func (s *Store) Close() error {
 	return nil
 }
 
+// Refresh synchronously checks for new versions of all the secrets currently
+// known by s. It blocks until the refresh is complete or until ctx ends.
+//
+// Updates are managed automatically when a Store is created and by the polling
+// mechanism, but a caller may invoke Refresh directly if it wants to check for
+// new secret values at a specific moment.
+func (s *Store) Refresh(ctx context.Context) error {
+	s.countPolls.Add(1)
```

Comment:
Should refreshes be x/singleflighted? With this API, a naively written client could generate a flurry of server work (and audit logs?) for something that could be coalesced gracefully.

Reply:
Not a bad idea. I figured a client could do that anyway, if they just grab the client and spam the server whenever they want a new secret value, but there wouldn't be any harm in adding a bit of extra politeness here. Will do.